dwave_networkx/drawing/chimera_layout.py

```diff
@@ -153,10 +172,11 @@ def chimera_node_placer_2d(m, n, t, scale=1., center=None, dim=2):
     else:
         center = np.asarray(center)
 
-    paddims = dim - 2
-    if paddims < 0:
+    if dim < 2:
         raise ValueError("layout must have at least two dimensions")
 
+    paddims = np.zeros(dim - 2, dtype='float')
```

Comment:
Perhaps add a comment above with a short explanation of what paddim is used for?